scripts/Install-VSCode.ps1

```diff
@@ -0,0 +1,111 @@
+<#
+.SYNOPSIS
+    Installs Visual Studio Code, the PowerShell extension, and optionally
+    a list of additional extensions.
+
+.DESCRIPTION
+    This script can be used to easily install Visual Studio Code and the
+    PowerShell extension on your machine.  You may also specify additional
+    extensions to be installed using the -AdditionalExtensions parameter.
+    The -LaunchWhenDone parameter will cause VS Code to be launched as
+    soon as installation has completed.
+
+    Please contribute improvements to this script on GitHub!
+
+    https://github.com/PowerShell/vscode-powershell/blob/master/scripts/Install-VSCode.ps1
+
+.PARAMETER AdditionalExtensions
+    An array of strings that are the fully-qualified names of extensions to be
+    installed in addition to the PowerShell extension.  The fully qualified
+    name is formatted as "<publisher name>.<extension name>" and can be found
+    next to the extension's name in the details tab that appears when you
+    click an extension in the Extensions panel in Visual Studio Code.
+
+.PARAMETER LaunchWhenDone
+    When present, causes Visual Studio Code to be launched as soon as installation
+    has finished.
+
+.EXAMPLE
+    Install-VSCode.ps1 -LaunchWhenDone
+
+    Installs Visual Studio Code and the PowerShell extension and then launches
+    the editor after installation completes.
+
+.EXAMPLE
+    Install-VSCode.ps1 -AdditionalExtensions @('eamodio.gitlens')
+
+    Installs Visual Studio Code, the PowerShell extension, and the Git Lens
+    extension.
+
+.NOTES
+    This script is licensed under the MIT License:
+
+    Copyright (c) Microsoft Corporation.
+
+    Permission is hereby granted, free of charge, to any person obtaining a copy
+    of this software and associated documentation files (the "Software"), to deal
+    in the Software without restriction, including without limitation the rights
+    to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+    copies of the Software, and to permit persons to whom the Software is
+    furnished to do so, subject to the following conditions:
+
+    The above copyright notice and this permission notice shall be included in all
+    copies or substantial portions of the Software.
+
+    THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+    IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+    FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+    AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+    LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+    OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+    SOFTWARE.
+#>
+[CmdletBinding()]
+param(
+    [Parameter(Mandatory=$false)]
```

Comment:
I usually don't bother specifying Mandatory=$false since that is the default.  Other than that LGTM.